src/SourceBuild/content/build.proj

```diff
@@ -25,6 +27,22 @@
              StopOnFirstFailure="true" />
   </Target>
 
+  <Target Name="MergeAssetManifests" AfterTargets="Build">
+    <PropertyGroup>      
+      <MergedAssetManifestOutputPath>$(ArtifactsDir)VerticalManifest.xml</MergedAssetManifestOutputPath>
+      <VmrBuildNumber>$(BUILD_BUILDNUMBER)</VmrBuildNumber>
```

Comment:
Move this property after L7 in this file and see if that changes anything.

Reply:
should work there, before we go into repo-projects

Reply:
Yeah, no idea why it's not available now. Looks like we stopped publishing binlogs..

Reply:
I don't see this variable mentioned anywhere else in the code.. I reverted back to the Build.props change